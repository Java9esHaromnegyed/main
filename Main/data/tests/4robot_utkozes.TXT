loadArena testArena4.txt
initArena
robotMovement 1 RIGHT
tick 1
robotMovement 1 UP
tick 1
robotMovement 1 UP
tick 1
robotMovement 2 LEFT
tick 1
robotMovement 2 UP
tick 1
addCleanerRobot 3 0 0
tick 1
addCleanerRobot 4 0 4
tick 1
