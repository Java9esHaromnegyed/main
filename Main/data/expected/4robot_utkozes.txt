arenaLoaded src: testArena4.txt
robotAdded id: 1 name: "player1" pos: [0; 0]
robotAdded id: 2 name: "player2" pos: [0; 5]
obstacleAdded type: OIL pos: [1; 0]
obstacleAdded type: OIL pos: [1; 3]
arenaInited
robotTurnedLeft id: 1 dir: UP
robotSpeedUp id: 1 speed: 2
robotSpeedUp id: 1 speed: 3
robotTurnedRight id: 2 dir: DOWN
robotSpeedUp id: 2 speed: 2
tick
robotMoved id: 1 pos: [0; 3]
robotMoved id: 2 pos: [0; 3]
robotCollided id1: 1 id2: 2 atPos: [0; 3]
robotDied id: 2
tick
robotMoved id: 1 pos: [0; 4]
cleanerRobotAdded id: 3 pos: [0; 0]
cleanerRobotAdded id: 4 pos: [0; 1]
tick
cleanerRobotMoved id: 3 pos: [1; 0]
cleanerRobotMoved id: 4 pos: [0; 0]
cleanerRobotClean id: 3 pos: [1; 0]
tick
cleanerRobotMoved id: 4 pos: [1; 0]
robotCollided id1: 3 id2: 4 atPos: [1; 0]
cleanerRobotClean id: 3 pos: [1; 0]
tick
cleanerRobotMoved id: 4 pos: [0; 0]
cleanerRobotClean id: 3 pos: [1; 0]
tick
cleanerRobotMoved id: 4 pos: [0; 1]
obstacleRemoved type: OIL pos: [1; 0]